docs/intro/arch_overview/outlier.rst

```diff
@@ -98,12 +102,33 @@ action
   brought back into service.
 
 type
-  If ``action`` is ``eject``, species the type of ejection that took place. Currently this can
-  only be ``5xx``.
+  If ``action`` is ``eject``, specifies the type of ejection that took place. Currently this can be:
+  ``5xx`` and ``SuccessRate``.
```

Comment:
The second sentence doesn't make sense. Can you rewrite it? Also colon after be isn't needed.